src/CMakeLists.txt

```diff
@@ -135,17 +135,17 @@ set(Mapper_Common_SRCS
  
  compass.cpp
 
- file_format.cpp
- file_format_registry.cpp
- file_import_export.cpp
- file_format_native.cpp
- file_format_ocad8.cpp
- file_format_xml.cpp
- 
- fileformats/ocd_file_export.cpp
- fileformats/ocd_file_format.cpp
- fileformats/ocd_file_import.cpp
- fileformats/ocd_types.cpp
+ file_formats/file_format.cpp
+ file_formats/file_format_registry.cpp
+ file_formats/file_import_export.cpp
+ file_formats/file_format_native.cpp
+ file_formats/file_format_ocad8.cpp
+ file_formats/file_format_xml.cpp
+
+ file_formats/ocad/ocd_file_export.cpp
+ file_formats/ocad/ocd_file_format.cpp
+ file_formats/ocad/ocd_file_import.cpp
+ file_formats/ocad/ocd_types.cpp
```

Comment:
Given that the import/export of newer OCAD files consists of 6 files (excluding the ocad_types_* files) and they don't follow the file_format_* naming scheme of other formats I think keeping them in a separate directory makes sense.

Will change ocad->ocd, would be interested in knowing if there is there some specific reason for that spelling though?